src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultLanguageServerFeatureOptions.cs

```diff
@@ -8,14 +8,16 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     internal class DefaultLanguageServerFeatureOptions : LanguageServerFeatureOptions
     {
-        public override bool SupportsFileManipulation { get; } = true;
+        public override bool SupportsFileManipulation => true;
 
-        public override string ProjectConfigurationFileName { get; } = LanguageServerConstants.DefaultProjectConfigurationFile;
+        public override string ProjectConfigurationFileName => LanguageServerConstants.DefaultProjectConfigurationFile;
 
         public override string CSharpVirtualDocumentSuffix => ".ide.g.cs";
 
         public override string HtmlVirtualDocumentSuffix => "__virtual.html";
 
-        public override bool SingleServerCompletionSupport { get; } = false;
+        public override bool SingleServerCompletionSupport => false;
+
+        public override bool SingleServerRenameSupport => false;
```

Comment:
Actually how do you feel about having a general "single server" flag now that we proved out completion?

Reply:
I think as long as we have an understanding that _most_ features won't need individual flags I'm on board, and will make my PR easier when I get to it ;)

Reply:
Do you mean just use `SingleServerCompletionSupport` as the flag? or rename this to `SingleServerSupport`?

Reply:
Rename this rename flag to `SingleServerSupport` so we then have 2 flags, `SingleServerCompletionSupport` and `SingleServerSupport`